### ISSUE TITLE
documentation for unblessed_bool is incorrect/outdated

### ISSUE THREAD

**Issue by karenetheridge** [rurban/Cpanel-JSON-XS#163]
The documentation says:

```
       $json = $json->unblessed_bool ([$enable])
       $enabled = $json->get_unblessed_bool
               $json = $json->unblessed_bool([$enable])

           If $enable is true (or missing), then "decode" will return Perl non-object boolean variables (1 and 0) for JSON booleans ("true" and "false"). If $enable is false, then "decode" will return "Cpanel::JSON::XS::Boolean"
           objects for JSON booleans.
```

This is not correct:

1. blessed booleans are JSON::PP::Boolean objects now, not Cpanel::JSON::XS::Boolean
~~2. unblessed_bool defaults to false.~~

```
: $; perl -MData::Dumper -MCpanel::JSON::XS -wle'print Dumper(Cpanel::JSON::XS->new->unblessed_bool->decode("[true]"))'
$VAR1 = [
          1
        ];

: $; perl -MData::Dumper -MCpanel::JSON::XS -wle'print Dumper(Cpanel::JSON::XS->new->unblessed_bool(0)->decode("[true]"))'
$VAR1 = [
          bless( do{\(my $o = 1)}, 'JSON::PP::Boolean' )
        ];

: $; perl -MData::Dumper -MCpanel::JSON::XS -wle'print Dumper(Cpanel::JSON::XS->new->decode("[true]"))'
$VAR1 = [
          bless( do{\(my $o = 1)}, 'JSON::PP::Boolean' )
        ];

```

**Comment by Grinnz** [comment]
The "or missing" does not indicate the default, it indicates that it will set the option true if you call it without passing a value (like the others).

**Comment by rurban** [comment]
Oops, thanks.

**Comment by karenetheridge** [comment]
#164 actually closed this issue more than #163, oops.

**Comment by rurban** [comment]
Fixed with he PR #164
